### PR TITLE
[AKS] `az aks create/update`: Update the VM SKU regex validation to include larger set of VMs for Azure Container Storage

### PR DESCRIPTION
**Pull request by mukhoakash**
**Related command**

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

File: src/azure-cli/azure/cli/command_modules/acs/azurecontainerstorage/_helpers.py [hunk 1]
```diff
@@ -414,7 +414,7 @@ def get_desired_resource_value_args(
 # Returns -1 if there is a problem with parsing the vm_size.
 def get_cores_from_sku(vm_size):
     cpu_value = -1
-    pattern = r'standard_([a-z]+)(\d+)([a-z]*)_v(\d+)'
+    pattern = r'standard_([a-z]+)(\d+)([a-z]*)(?:_[^_]+)*_v(\d+)'
```
Comment (FumingZhang) [inline src/azure-cli/azure/cli/command_modules/acs/azurecontainerstorage/_helpers.py:417]: Could you please add some unit test for the change?

**Comment by zhoxing-ms** [comment]
Please note that we are launching the release for this sprint this week. Please resolve all comments by tomorrow, otherwise the release of this PR will have to be postponed to the next sprint **(on 10-08)**

File: src/azure-cli/azure/cli/command_modules/acs/azurecontainerstorage/_helpers.py (hunk 1 above)
Comment (mukhoakash) [reply src/azure-cli/azure/cli/command_modules/acs/azurecontainerstorage/_helpers.py]: added

**Comment by mukhoakash** [comment]
@FumingZhang  @zhoxing-ms All the comments are resolved and CIs are fixed. Please review and sign off for the current milestone. Thank you

**Review by FumingZhang — APPROVED** [review]
LGTM

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**